lib/secure_cookies.rb

```diff
@@ -0,0 +1,29 @@
+class SecureCookies
+  COOKIE_SEPARATOR = "\n".freeze
+  SECURE_COOKIE_ATTRIBUTES = ['; Secure', '; HttpOnly', '; SameSite=Lax'].freeze
+
+  def initialize(app)
+    @app = app
+  end
+
+  def call(env)
+    status, headers, body = @app.call(env)
+
+    if headers['Set-Cookie'].present?
+      cookies = headers['Set-Cookie'].split(COOKIE_SEPARATOR)
+
+      cookies.each do |cookie|
+        attributes = SECURE_COOKIE_ATTRIBUTES.reject { |attr| attr if cookie.match?(/#{attr}/i) }
```

Comment:
1. I think we can drop the `attr if` part, because the reject just cares about true/false?
1. I think this is going to recompile each regex for each attribute on each request. Maybe we can make the regexes in advance as constants? 

```
SECURE_COOKIE_REGEXES = SECURE_COOKIE_ATTRIBUTES.map { |attr| /#{attr}/i }

attributes = SECURE_COOKIE_ATTRIBUTES.zip(SECURE_COOKIE_REGEXES).reject do |_attr, regex| cookie.match?(regex) }.map(&:first)
```